workers/analyzer/src/main/kotlin/analyzer/AnalyzerWorker.kt

```diff
@@ -80,8 +80,12 @@ internal class AnalyzerWorker(
             ortRun.path.orEmpty()
         )
 
-        val resolvedEnvConfig = envConfigFromJob?.let { environmentService.setUpEnvironment(context, it) }
-            ?: environmentService.setUpEnvironment(context, sourcesDir, repositoryService)
+        val resolvedEnvConfig = environmentService.setUpEnvironment(
```

Comment:
Commit message:
* Typo in "AnalyzerJobConfiguration"
* Say  "...to merge the `EnvironmentConfig` from the `.ort.env.yml` file in the repository with..." to provide a bit more context.
* Explain if entries from the `.ort.env.yml` or from the job configuration have priority.